Libplanet.Net.Tests/Consensus/Context/ContextNonProposerTest.cs

```diff
@@ -47,23 +47,18 @@ public async Task EnterPreVoteBlockOneThird()
                     stateChangedToRoundOnePreVote.Set();
                 }
             };
-            context.Start();
 
+            context.Start();
             context.ProduceMessage(
                 TestUtils.CreateConsensusPropose(
                     block, TestUtils.Peer1Priv, round: 0));
```

Comment:
A bit confusing, do we need this line?

Reply:
Just some old artifact I suppose. I don't think we really need it. 😗